Project-I-Social-Card/social-card/src/components/CardComponents/CardContainer.js

```diff
@@ -1,2 +1,32 @@
-import React from 'react';
-import './Card.css';
+import React, { Component } from "react";
+import "./Card.css";
+import CardContent from "./CardContent";
+const data = {
+  summary: {
+    title: "Get started with React",
+    summaryText: `React makes it painless to create interactive UIs. Design simple views for
+    each state in your application.`,
+    link: "https://www.reactjs.org",
+    linkText: "reactjs.org"
+  }
+};
+
+class CardContainer extends Component {
+  handleClick = () => {
+    window.location.href = "https://reactjs.org";
+  };
```

Comment:
I'm not sure why you would use this instead of just using an ```href``` unless I'm missing something?!

Reply:
Uhm actually I didn't think it was a good idea to wrap the entire component in an anchor tag. Would that be okay?